templates/includes/record-matters.html

```diff
@@ -0,0 +1,61 @@
+{% load records_tags wagtailcore_tags %}
+<div class="generic-intro record-matters">
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-md-10 col-sm-12">
+                <h2 class="story-title">The story of this record</h2>
+                {% if record.date_created or record.reference_number %}
+                    <div class="record-matters__content">
+                        {% if record.date_created %}
+                            <p class="record-matters__text">
+                                Date: <strong>{{ record.date_created }}</strong>
+                            </p>
+                        {% endif %}
+                        {% if record.reference_number %}
+                            <p class="record-matters__text">
+                                Catalogue reference: <a class="record-matters__text record-matters__text--bold"
+    href="{% record_url record is_editorial=True %}">{{ record.reference_number }}</a>
+                            </p>
+                        {% endif %}
+                    </div>
+                {% endif %}
+                {% if record.description %}
+                    <div id="readLessContent" class="hidden">
+                        {{ record.description|richtext|truncatewords_html:50 }}
+                        <button id="readMoreButton" class="record-matters__plain-button" href="">
+                            Read more
```

Comment:
I think this could be more descriptive, e.g. 'Expand to read more about {{ name }}' or similar - if that's too long visually we could add it as sr-only text or an aria-label

Reply:
Good point, added a label.